react/features/unsupported-browser/components/__tests__/UnsupportedMobileBrowser.spec.js

```diff
@@ -0,0 +1,93 @@
+/* @flow */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import HideNotificationBarStyle from '../HideNotificationBarStyle';
+import { UnsupportedMobileBrowser } from '../UnsupportedMobileBrowser';
+
+const oldLocation = window.location;
+
+declare var afterAll: Function;
+declare var beforeAll: Function;
+declare var describe: Function;
+declare var expect: Function;
+declare var it: Function;
+
+/**
+ * Renders UnsupportedMobileBrowser with corresponding props.
+ *
+ * @param {string} room - Room name proprety.
+ * @returns {ShallowWrapper}
+ * @private
+ */
+function _prepareComponent(room) {
+    const props = {
+        _room: room,
+        t: key => key
+    };
+
+    return shallow(<UnsupportedMobileBrowser { ...props } />);
+}
+
+/**
+ * Test suite related to UnsupportedMobileBrowser component.
+ */
+describe('UnsupportedMobileBrowser component', () => {
+
+    /**
+     * Since tests are being run in Node environment we should set some browser
+     * specific data like location etc before running the tests.
+     */
+    beforeAll(() => {
+        window.location = window.location || {
+            href: 'https://test'
+        };
+    });
+
+    /**
+     * Set previous value of location after running of all the tests.
+     */
+    afterAll(() => {
+        window.location = oldLocation;
```

Comment:
Sure, we can think of stubbing such things. This PR is like proof of concept. And of course if it's a thing that jitsi-meet need, we can go forward and think about stubs fixtures and so on.